tests/rptest/utils/si_utils.py

```diff
@@ -465,11 +472,25 @@ def manifest_for_ntp(self,
         self.logger.debug(f'manifest: {pprint.pformat(manifest_data)}')
         return manifest_data
 
+    def cloug_log_segment_count_for_ntp(self,
+                                        topic: str,
+                                        partition: int,
+                                        ns: str = 'kafka') -> int:
+        manifest = self.manifest_for_ntp(topic, partition, ns)
+        if 'segments' not in manifest:
+            return 0
+
+        return len(manifest['segments'].values())
```

Comment:
could this be simplified to: `return len(manifest['segments'])`

Reply:
Fixed.